app/styles/pages/public-event.scss

```diff
@@ -185,3 +185,7 @@
 .event-map > iframe.g-map {
   height: 300px;
 }
+
+button.ui.blue.button.right.labeled.icon.small.right.aligned {
+  margin-top: 5px;
+}
```

Comment:
Can you fix it using semantic UI classes or predefined classes?

Reply:
@snitin315 but this bug is on mobile devices

Reply:
@snitin315 the bug in mobile view not the desktop site

Reply:
`device.isMobile`

Reply:
instead of margin-top use mt-8 class